packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -536,12 +536,12 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
         selectAll?: boolean
     ) {
         const range = editor.getDocument().createRange();
+        // Get deepest editable position in the cell
+        const { node, offset } = normalizePos(cell, nodeOffset);
+
         if (selectAll) {
-            range.selectNodeContents(cell);
+            range.selectNodeContents(node);
```

Comment:
So if there are multiple nodes in next cell, it will only select the first one. Is that expected?

![image](https://github.com/user-attachments/assets/33f94a22-7c0a-4ac3-b554-164c3ccab5ab)

Reply:
![image](https://github.com/user-attachments/assets/aeeb6aaa-cdac-407f-be6a-cf7cd1f79d56)

In this example, there are Bold style in middle of the text, which causes 3 text nodes existing. This change will cause only the first part is selected.

Reply:
Good point. The specific scenario this is trying to fix is one where the cell is empty and only has a div with br inside. I could just look for that scenario specifically, but I am not sure if it is reliable to hardcode it that way.

Could this be an issue with text input and not selection?

Reply:
If you expect different behavior between empty cell and cell with content, then yes you need to have different code for them.